packages/testing/src/chai-retry-plugin/helpers.ts

```diff
@@ -1,9 +1,22 @@
 import Chai from 'chai';
-import { timeout as timeoutPromise, sleep } from 'promise-assist';
+import { timeout as timeoutPromise, sleep as sleepPromise } from 'promise-assist';
```

Comment:
it's a matter of opinion of course, but in such cases I'd not destructure, but instead did this:
```
import promiseHelpers from 'promise-assist';
...
promiseHelpers.sleep();
```

Also `sleepPromise` is a bad name, because it's function that returns promise - not promise.

Reply:
@yurii-ve nice idea, fixed